### ISSUE TITLE
Flat disc instead of hierarchy

### ISSUE THREAD

**Issue by blup** [hughsk/disc#31]
Maybe I'm doing something wrong, but I seem to only get a flat disc instead of a hierarchical view. I'm running the following command from gulp:
```
  browserify({ extensions: ['.coffee', '.js'], entries: [__dirname + '\\client\\js\\app.coffee'], fullPaths: true })
    .transform(coffeeify, {global: true})
    .bundle()
    .pipe(disc())
    .pipe(fs.createWriteStream(output))
```
![image](https://cloud.githubusercontent.com/assets/437915/5993550/303ae4ae-aa12-11e4-8e06-17adb0c9eece.png)

**Comment by pH200** [comment]
It's because you're running on Windows. This should be fixed by this patch hughsk/file-tree#1

**Comment by blup** [comment]
Thanks! That worked perfectly.
